src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Refactoring/RenameEndpoint.cs

```diff
@@ -118,32 +144,7 @@ internal class RenameEndpoint : IRenameEndpoint
                 }
             }
 
-            // If we're not doing single server rename, then we're done. C# and Html will be handled by the RenameHandler in the HtmlCSharp server.
```

Comment:
I can't comment on lines not in the PR, but line 138 has a check for language type and does `TryGetRazorComponentRenameEditsAsync` instead of doing a delegated pass.